modules/openapi-generator/src/main/resources/scala-akka-client/model.mustache

```diff
@@ -1,9 +1,10 @@
 {{>licenseInfo}}
 package {{package}}
 
+{{#imports}}
+import {{import}}
+{{/imports}}
 import {{mainPackage}}.core.ApiModel
-import org.joda.time.DateTime
-import java.util.UUID
```

Comment:
This will be a potential breaking change unless this is added to the constructor:

```
importMapping.put("UUID", "java.util.UUID");
```

Reply:
it is already there in `ScalaAkkaClientCodegen extends AbstractScalaCodegen extends` [DefaultCodegen](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L1229) base class.
I will add unit test to check there is no breaking changes for the further versions